styles.css

```diff
@@ -41,8 +41,8 @@
     margin-bottom: 10px;
 }
 
-.turnitinsim_eulacontainer #pp-eula-accept {
-    margin-left: 20px;
+.eulacontainer #pp-eula-accept {
```

Comment:
This should be done the opposite way. We need to keep the turnitinsim_ prefix as that is Moodle's style guidelines. Instead you should change where `eulacontainer` is being used.